frontend/src/components/Header.tsx

```diff
@@ -330,7 +327,7 @@ const useStyles = makeStyles((theme) => ({
   menuButton: {
     marginRight: theme.spacing(2),
     display: 'block',
-    [theme.breakpoints.up('lg')]: {
+    [theme.breakpoints.up(1330)]: {
```

Comment:
See if we can define something like `lg` / `md` if that's easy -- otherwise, let's just keep this as it is.